.gitignore

```diff
@@ -1,13 +1,86 @@
-# compile output
+# Git
```

Comment:
I'm against adding too much things in .gitignore, after all the project is not that big. If someone's configuration generates weird files, they can just add those if needed.

Reply:
roger